docs/release_docs/AHA-Prod_ReleaseFlow.md

```diff
@@ -0,0 +1,29 @@
+# AHA Prod Release Workflow
+Explaination of AHA Prod Release Workflow.
+![AHA-PROD](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/AHA-PROD_Workflow.png?raw=true)
+![AHA-PROD-JOB](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/aha_workflow.png?raw=true)
+
+## AHA-PROD-CI-CD
```

Comment:
This header and the brief explanation should come above the architecture diagrams